go/billing/tasks.py

```diff
@@ -561,6 +561,7 @@ def set_account_balance(account_number, balance):
     """
     account = Account.objects.get(account_number=account_number)
     credit_amount = Decimal(str(balance)) - account.credit_balance
+    credit_amount = max(Decimal('0.0'), credit_amount)
```

Comment:
I thought it might be useful to see that the task actually ran, and credited with `0.0`, than not knowing if it was because the account had enough credits, or if the task failed. Happy to change it though.